General/general_shacl.ttl

```diff
@@ -0,0 +1,133 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+general:GeneralShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node gx:DataResource ;
+            sh:order 1 ;
+            sh:path general:DataResource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:LinksShape ;
+            sh:order 4 ;
+            sh:path general:Links ] ;
```

Comment:
attributes should be defined as camelCase --> `general:links`

Reply:
fix camelCase path of main categories